.circleci/verbatim-sources/workflows-docker-builder.yml

```diff
@@ -0,0 +1,59 @@
+  docker_build:
+    jobs:
+      - docker_build_job:
+          name: "pytorch-linux-bionic-clang9-thrift-llvmdev"
+          image_name: "pytorch-linux-bionic-clang9-thrift-llvmdev"
```

Comment:
Bah, what horrible design on CircleCI's part XD

Reply:
yep, tried using parameters passed in, no lucky